project/src/pages/main-page/main-page.tsx

```diff
@@ -0,0 +1,46 @@
+import FilmsList from '../../components/films-list/films-list';
+import GenresList from '../../components/genres-list/genres-list';
+import Logo from '../../components/logo/logo';
+import PromoFilmCard from '../../components/promo-film-card/promo-film-card';
+import { Films } from '../../types/film';
+import { FILMS_AMOUNT} from '../../utils/constants';
+
+type MainPageProps = {
+  films:Films;
+}
+
+function MainPage({films}:MainPageProps):JSX.Element{
+  const genres = ['All genres'];
```

Comment:
const genres = [`'All genres'`];

https://github.com/htmlacademy-react/1984913-what-to-watch-12/blob/053e0cfa5bb1c534e2dfe88bbfc216c67dd2352f/project/src/utils/constants.ts#L5